apps/remix-dapp/src/locales/en/udapp.json

```diff
@@ -63,6 +65,7 @@
   "udapp._comment_environment.tsx": "libs/remix-ui/run-tab/src/lib/components/environment.tsx",
   "udapp.environment": "Environment",
   "udapp.environmentDocs": "Click for docs about Environment",
+  "udapp.saveVmState": "Save VM state",
```

Comment:
this one is already declared above.

Reply:
This one is for tooltip. above one is for modal title. I kept them separate, in case different text is needed at both places.